src/Dotnet.Installer.Console/Commands/ListCommand.cs

```diff
@@ -66,8 +104,10 @@ string ComponentStatus(string key, int majorVersion)
                     else
                     {
                         status = available
-                            ? "[bold blue]Available[/]"
-                            : "[bold grey]Unavailable[/]";
+                            ? isEol
+                                ? "Available"
+                                : "[bold blue]Available[/]"
+                            : "[bold grey]-[/]";
                     }
 
                     return status;
```

Comment:
**suggestion:** 3 layers of if/else is not nice to read. Maybe use the early-return pattern.

```csharp
                    if (components.TryGetValue(key, out var component))
                    {
                        return $"[bold green]Installed [[{component.Version}]][/]";
                    }

                    var available = _manifestService.Remote.Any(c => c.Name == key && c.MajorVersion == majorVersion);

                    if (!available)
                    {
                        return "[bold grey]-[/]";
                    }

                    if (isEol)
                    {
                        return "Available";
                    }

                    return "[bold blue]Available[/]";
```

*note: I can not use the GitHub suggestion tool, because there are line that do not belong to the diff :/*